isvcs/zk.go

```diff
@@ -19,13 +19,26 @@ import (
 	"io/ioutil"
 	"net"
 	"time"
+	"github.com/control-center/serviced/config"
+	"fmt"
 )
 
 var Zookeeper IServiceDefinition
 var zookeeper *IService
 
+const QUORUM_HEALTHCHECK_NAME = "Participating in Quorum"
```

Comment:
Change made as requested.  When shown in the UI, the "hasQuorum" format fits nicely with the existing "running" check.